covertest.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python
```

Comment:
Was this written for this project or copied from elsewhere? Do we really want to start requiring python for this to work?

I only see this as future technical debt.